docs/FAQ

```diff
@@ -0,0 +1,44 @@
+# Redocly Frequently Asked Questions
+
+- [How do I enable the Console API?](#how-do-i-enable-the-console-api)
+- [Redoc Community Edition is not rendering my large json file smoothly.](#redoc-community-edition-is-not-rendering-my-large-json-file-smoothly)
+- [Does Redocly support webhooks with OpenAPI 3.1?](#does-redocly-support-webhooks-with-openapi-31)
+- [How do I adjust the theme settings on a page, such as color, line spacing, and fonts?](#how-do-i-adjust-the-theme-settings-on-a-page-such-as-color-line-spacing-and-fonts)
+- [Where can I locate Docker images for Redocly and redoc.cli?](#where-can-i-locate-docker-images-for-redocly-and-redoc-cli)
+- [Which tag should I use for operations that cannot be logically grouped?](#which-tag-should-i-use-for-operations-that-cannot-be-logically-grouped)
+- [Which vendor extensions does Redocly support?](#which-vendor-extensions-does-redocly-support)
+
+## How do I enable the Console API?
+
+Redoc.ly API Reference Docs supports the Console API, though it is disabled by default. You can enable it in 'API Reference Settings'->'Live Configuration'.
+
+## Redoc Community Edition is not rendering my large json file smoothly.
+
+Consider upgrading. Redocly API Reference Docs includes an option to section files.
+
+'referenceDocs:
```

Comment:
The code sample should be in a fenced code block:

https://www.markdownguide.org/extended-syntax/#fenced-code-blocks

Reply:
Well, that's frustrating. I did add the 4 spaces, but they must have been lost on copy+paste. Readding.

Reply:
Done